src/components/InteractionPage/InteractionPage.test.js

```diff
@@ -10,3 +13,68 @@ test("The Interaction Page renders", () => {
     </Router>
   );
 });
+
+test("Game starts from scratch when user re-enters page", () => {
+  const { getByText } = render(
+    <Router>
+      <InteractionPage id="1" />
+    </Router>
+  );
+  const answers = data.places[0].interactions[0].answers;
+  answers.map(answer =>
+    expect(getByText(answer.text).className).toBe("option")
+  );
+});
+
+
+test("Selecting an answer changes the class to 'selected'", () => {
+  const setCompleted = jest.fn();
+  const { getByText } = render(
+    <Router>
+      <InteractionPage id="1" setCompleted={setCompleted} />
+    </Router>
+  );
+  const answers = data.places[0].interactions[0].answers;
+
+  answers.map(answer => {
+    const oneAnswer = getByText(answer.text);
+    fireEvent.click(oneAnswer);
+    if (answer.correct) {
+      expect(oneAnswer.className).toBe("selected");
+    } else {
+      expect(oneAnswer.className).toBe("option");
+    }
+  });
+});
+
+test("Test if there is a back button with correct link'", () => {
+  const { getByText } = render(
+    <Router>
+      <InteractionPage id="1" />
+    </Router>
+  );
+  const backButton = getByText("Go Back");
+  expect(backButton.href).toMatch(/0/);
```

Comment:
It's not specific enough, it will just check it has digit 0, maybe have

Reply:
💩